docs/README.md

```diff
@@ -26,3 +26,5 @@ Rest of the information you can find in those sections
 * [Debugging](debugging.md) - useful debugging information.
 * [Unit tests](unit_tests.md) - how to run and write unit tests.
 * [Workarounds](workarounds.md) - how to manage workarounds in ocs-ci.
+* [Using ocs-ci to deploy OCP cluster without default OCS
```

Comment:
Do you think it should be that general section mentioned in our first level of documentation? What about have section for config files and their usage? Or can be under usage section.

Reply:
I placed this into a separate doc file as this is not directly about general usage of ocs-ci, nor about a particular ocs-ci option, but about a particular use case. I believe it would be good for us to have few common or important scenario documented in this FAQ  like, use case style.

I definitely don't want to duplicate anything which is already discussed with detail elsewhere, such as how to use configuration or configure ocs-ci. It would make sense to reference the general docs here, but I'm not sure how to do that with markdown.

Reply:
What I meant is to have similar file like this current one when you updating with name like:

`docs/specific_ocs_ci_use_cases.md`

Where you will have just content like we have here with link to deployment_without_ocs.md + other use cases can be linked from that place as well. So we will have one general MD file with all the specific use cases described in short way and not have them directly mentioned in the first level.

Reply:
This is a good idea.

That said, I would rather keep each usecase in a separate file. From the indexing perspective, you would just reference it via usecase/* in sphinx toc directive.

Reply:
@petr-balogh do you want me to do this in this pull request?

Reply:
can be next one ;) noticed to late anyway.. Thanks